rs/backend/src/accounts_store.rs

```diff
@@ -781,8 +781,10 @@ impl AccountsStore {
     }
 
     /// Gets all the transactions to or from the caller.
+    /// TODO: Delete this.
```

Comment:
Nit. Would it make sense to do the following to track the potential leftover work?

```suggestion
    /// TODO[NNS1-ticket-number]: Delete this.
```

Reply:
Done